ZenPacks/zenoss/OpenStackInfrastructure/zenpack.yaml

```diff
@@ -129,6 +129,12 @@ zProperties:
       Hold any update for at most this long before processing.  Any other
       updates to the same object during this time window will be combined
       into a single zenoss model update.
+  zOpenStackUserDomainName:
+    default: 'default'
```

Comment:
These need more documentation, both here and in the zenpack docs.  We probably need explanation of
- what an openstack domain is
- what the user and project domains are (how they differ)
- how to choose the right one to use
- what user access role we require, and how/if that relates to the settings